### ISSUE TITLE
Nightly - 6.5 - 5-22-NFS-Volume - Authentication failed for user root

### ISSUE THREAD

**Issue by rajanashok** [vmware/vic#7641]
```00.001KEYWORD ${strippedPW} = String . Remove String %{DEPLOYED_PASSWORD}, \\
Documentation:	
Removes all removables from the given string.

Start / End / Elapsed:	20180404 04:47:10.370 / 20180404 04:47:10.371 / 00:00:00.001
04:47:10.371	INFO	${strippedPW} = '<stripped by hickeng manually>'	
00:00:00.000KEYWORD SSHLibrary . Open Connection ${NFS_IP}
Documentation:	
Opens a new SSH connection to the given `host` and `port`.

Start / End / Elapsed:	20180404 04:47:10.372 / 20180404 04:47:10.372 / 00:00:00.000
00:02:12.307KEYWORD BuiltIn . Wait Until Keyword Succeeds 2 min, 30 sec, Login, root, ${strippedPW}
Documentation:	
Runs the specified keyword and retries if it fails.

Start / End / Elapsed:	20180404 04:47:10.373 / 20180404 04:49:22.680 / 00:02:12.307
00:00:02.672KEYWORD SSHLibrary . Login root, ${strippedPW}
Documentation:	
Logs into the SSH server with the given `username` and `password`.

Start / End / Elapsed:	20180404 04:47:10.374 / 20180404 04:47:13.046 / 00:00:02.672
04:47:10.375	INFO	Logging into '10.192.242.63:22' as 'root'.	
04:47:13.046	FAIL	Authentication failed for user 'root'.	
```

https://storage.cloud.google.com/vic-ci-logs/vic_nightly_logs_2018-04-04-01-49-39.zip?authuser=1

**Comment by hickeng** [comment]
@mhagen-vmware looks like there was an issue with #7625

**Comment by mhagen-vmware** [comment]
This should be fixed with #7646
